### ISSUE TITLE
Regex test on challenge doesn't allow no semi-colon

### ISSUE THREAD

**Issue by moT01** [freeCodeCamp/freeCodeCamp#43536]
From chat:

> Hello everyone, I found a small problem with this question. When I add `margin: auto` in the last line, it must add ";", otherwise I cannot pass the question (but browsing works). Does it need to be resolve?

> The question: https://www.freecodecamp.org/learn/responsive-web-design/applied-visual-design/center-an-element-horizontally-using-the-margin-property

It's a minor edge case, I wouldn't object to a fix. The file is [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/01-responsive-web-design/applied-visual-design/center-an-element-horizontally-using-the-margin-property.md). I suppose the regex would need to check if it ends with `;` or some space and `}`. Also, there's no check making sure the property is in the div. We could probably add a check for that while we're at it.

**Comment by ShaunSHamilton** [comment]
Prime opportunity to use [CSSHelp](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/utils/css-help.ts)

An issue with the _help wanted_ or _first timers only_ label is open for contribution. The first comprehensive PR created will be reviewed and merged. We typically do not assign issues to anyone other than long-time contributors.

If you would like to contribute, and have not read the contributors docs, please do so here: https://contribute.freecodecamp.org/#/

If you have any issues with contributing, be sure to join us on the [contributors channel](https://chat.freecodecamp.org/channel/contributors), or on the [contributors sub-forum](https://forum.freecodecamp.org/c/contributors/3)

**Comment by jamesgeer** [comment]
Hi @moT01, I've opened a pr regarding this issue.

I tried implementing a replacement code assessment to replace the regex, but the test cases always failed. (for me atleast).

For example, I tried:

    assert($('div').css('margin') == 'auto');

Is there a reason this works in [this file](https://github.com/freeCodeCamp/freeCodeCamp/blob/73e5c4d935578e76c01e0248a7e6d042c99fb52b/curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-visual-balance-using-the-text-align-property.md) and not the issue file?

(Sorry if I'm missing something obious!)
